planner/core/point_get_plan.go

```diff
@@ -84,6 +84,14 @@ func (p *PointGetPlan) Schema() *expression.Schema {
 	return p.schema
 }
 
+// Cost implements PhysicalPlan interface
+func (p *PointGetPlan) Cost() float64 {
+	return 0
+}
+
+// SetCost implements PhysicalPlan interface
+func (p *PointGetPlan) SetCost(cost float64) {}
```

Comment:
Why implement them this way, will this make costs of point/batch get plans always be 0?

Reply:
Yeah, because I don't know where to set this cost, could you point it out for me?

Reply:
I advise to add this logic in `tryWhereIn2BatchPointGet`, `tryPointGetPlan` and `DataSource.convertToPointGet`, and you can use `Batch/PointGet.GetCost` to get their costs.

Reply:
added, but in `tryWhereIn2BatchPointGet` and `tryPointGetPlan`, there is no estimated cost, becuase we don't need to  compare its cost with other operators.

Reply:
OK, in that case, leaving the cost be 0 is OK for me.